kikit/panelize.py

```diff
@@ -613,7 +614,7 @@ def transferProjectSettings(self):
         into the project file.
         """
         if len(self.sourcePaths) > 1:
-            raise RuntimeError("Merging of DRC rules of multiple boards is currently unsupported")
+            print("WARNING: Multiple source boards found. DRC Rules MUST MATCH. Unpredictable behavior if not.")
```

Comment:
Please, never print in non-UI code. It prevents the reusability of the code. Also, just printing a warning is against the spirit of KiKit - you want to be able to trust the panels it produces. Just saying, "hey, this might turn out bad," is wrong. We should terminate on dangerous operations and add an option to **explicitly** enabling for dangerous stuff.